lib/chitter.rb

```diff
@@ -0,0 +1,22 @@
+class Chitter
+  attr_reader :id, :peep, :time
+
```

Comment:
Maybe use 'Peep' as the class name to clearly show what object this class is responsible for

Reply:
Agree. Can't do it here but learnt a lesson for future